[2024-04-20T15:28:58.581+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-20T15:28:58.604+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag.first_task scheduled__2023-10-02T00:00:00+00:00 [queued]>
[2024-04-20T15:28:58.612+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag.first_task scheduled__2023-10-02T00:00:00+00:00 [queued]>
[2024-04-20T15:28:58.613+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 6
[2024-04-20T15:28:58.622+0000] {taskinstance.py:2327} INFO - Executing <Task(BashOperator): first_task> on 2023-10-02 00:00:00+00:00
[2024-04-20T15:28:58.628+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=804) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-04-20T15:28:58.628+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag', 'first_task', 'scheduled__2023-10-02T00:00:00+00:00', '--job-id', '67', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpvn1tsw5t']
[2024-04-20T15:28:58.631+0000] {standard_task_runner.py:91} INFO - Job 67: Subtask first_task
[2024-04-20T15:28:58.630+0000] {standard_task_runner.py:63} INFO - Started process 811 to run task
[2024-04-20T15:28:58.663+0000] {task_command.py:426} INFO - Running <TaskInstance: our_first_dag.first_task scheduled__2023-10-02T00:00:00+00:00 [running]> on host e87e2f3305bd
[2024-04-20T15:28:58.724+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='julionevadod' AIRFLOW_CTX_DAG_ID='our_first_dag' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2023-10-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-10-02T00:00:00+00:00'
[2024-04-20T15:28:58.726+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-20T15:28:58.735+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-04-20T15:28:58.738+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hello world, this is the first task!']
[2024-04-20T15:28:58.743+0000] {subprocess.py:86} INFO - Output:
[2024-04-20T15:28:58.744+0000] {subprocess.py:93} INFO - hello world, this is the first task!
[2024-04-20T15:28:58.744+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-04-20T15:28:58.744+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-20T15:28:58.755+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=our_first_dag, task_id=first_task, execution_date=20231002T000000, start_date=20240420T152858, end_date=20240420T152858
[2024-04-20T15:28:58.773+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-20T15:28:58.781+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-04-20T15:28:58.781+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
